command/server_force_leave.go

```diff
@@ -28,7 +28,13 @@ Usage: nomad server force-leave [options] <node>
 
 General Options:
 
-  ` + generalOptionsUsage(usageOptsDefault|usageOptsNoNamespace)
+  ` + generalOptionsUsage(usageOptsDefault|usageOptsNoNamespace) + `
+  
+Server Force-Leave Options:
+
+  -prune
+    Removes failed or left server from the list of members entirely.
```

Comment:
I'm not sure words like "entirely" or "completely" (as in the help text below) sufficiently describe the difference between what happens here. Let's be much more specific and alter the main text to say "the failed or left server is marked as removed, and will be garbage collected after 24h" and then here "the failed or left server is removed from the Serf membership immediately." (This same suggestion applies everywhere we have docs changes in this PR.)

Also, in the above docs, it says:
> Note that if the member is actually still alive, it will eventually rejoin the cluster again.

I think that still applies here, right? So we should make doubly-sure it's clear that we're removing it from the current list of Serf members but that a live server will still rejoin.

Reply:
Thank you! I have updated the wording to reiterate that `-prune` won't work with alive server as well as added the part of node being garbaged collected after 24h.